src/pages/AddReservation.js

```diff
@@ -1,19 +1,29 @@
 import React from 'react';
+
 import ReservationForm from '../components/ReservationForm';
+import doctorsDefault from '../assets/doctors-form-bg.jpg';
+
+const backgroundStyle = {
+  background: `url(${doctorsDefault}) no-repeat center center fixed`,
+  backgroundSize: 'cover',
+  backgroundColor: '#afd917',
+  opacity: '0.7',
+  backgroundBlendMode: 'multiply',
```

Comment:
Try to not use constants in the styles, this is a **bad practice**

Reply:
Done! 😄